lib/module/doom.go

```diff
@@ -0,0 +1,63 @@
+package module
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/dvdmuckle/irkbot/lib/configure"
+	"github.com/dvdmuckle/irkbot/lib/message"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type doomStruct struct {
+	Type string `json:"type"`
+}
+
+var doomHost string
+var doomValids = []string{"shoot", "forward", "backward", "left", "right", "use"}
+
+func ConfigDoom(cfg *configure.Config) {
+	doomHost = cfg.Modules["doom"]["doom_host"]
+}
+
+func HelpDoom() []string {
+	s := "doom <command> - play doom!"
+	return []string{s}
+}
+
+func Doom(cfg *configure.Config, in *message.InboundMsg, actions *Actions) {
+	if len(in.MsgArgs[1:]) == 0 {
+		actions.Say("enter a command, dipstick")
+		return
+	}
+	doomCommand := strings.Join(in.MsgArgs[1:], " ")
+	doomValid := false
+	for i := 0; i < 6; i++ {
```

Comment:
Surely you didn't just hardcode the length of an array :P

Please fix this for loop to use the range construct: https://stackoverflow.com/a/7782507